### ISSUE TITLE
Highlighting like ACE Cloud9

### ISSUE THREAD

**Issue by xpepermint** [atom/language-ruby-on-rails#28]
I started the descusion [here](https://discuss.atom.io/t/rails-highlighting-like-ace-cloud9/14771). The code is much more readable in [ACE](https://github.com/ajaxorg/ace). In the example bellow the `if:`, `unless:`, `presente:` should be colored differently (now with the same color).

```ruby
class Product < ActiveRecord::Base
  belongs_to :project

  enum status: { enabled: 1, disabled: 2 }

  validates :project, :type, presence: true
  with_options if: :something?, unless: :something? do |step|
    step.validates :name, :description, presence: true
  end
end
```
Can we add this?

**Comment by lee-dohm** [comment]
I disagree. The `if:`, `unless:` and `presence:` in your example are all symbols and should be colored the same as other symbols, such as `status:`, `enabled:` and `disabled:` from your example. Unless there is some concrete and repeatable heuristic you can describe other than "symbols that have the same text as keywords"? (Though that wouldn't explain your suggestion that `presence:` be colored differently.)
